config/purgecss.config.js

```diff
@@ -4,4 +4,13 @@ export default {
   content: [`${paths.dist}/*.html`],
   keyframes: true,
   variables: true,
+  whitelistPatterns: [/^is|has-/],
+  extractors: [
+    {
+      extractor: (content) => {
+        return content.match(/[a-z-_\\@%0-9]+/g) || []; // Compare class names including '@', '%' and '\'
+      },
+      extensions: ['css', 'html'],
+    },
+  ],
```

Comment:
```suggestion
  whitelistPatterns: [/is-[\S]*/g, /has-[\S]*/g],
  // Allow to use special characters in class names like `@, :, %`
  defaultExtractor: (content) => content.match(/[\w-/:%@]+(?<!:)/g) || [],
```

Reply:
Don't we need to add \\\ to `/[\w-/:%@]+(?<!:)/g` -> `/[\w-\\/:%@]+(?<!:)/g`

Reply:
No, It will work in this way.

Reply:
I checked and it seems working in that way too, thank you 😄